cmd/adhoc/cmd/root.go

```diff
@@ -24,6 +28,30 @@ func init() {
 	)
 }
 
+func readFile() {
+	// Setting up file formatting
+	viper.SetConfigFile("syncron.yaml")
+	viper.SetConfigType("yaml")
+	viper.AddConfigPath(".")
+	// Reading from file
+	err := viper.ReadInConfig()
+	if err != nil {
+		logrus.Error("Config file not found.\n")
+		logrus.Info("Make sure syncron.yaml is in the correct folder. \n")
+		logrus.Info("Syncron looks for the config file on: \n")
+		logrus.Info("	- Root of the repository \n")
+		os.Exit(1)
+	} else {
+		logrus.Info("Your configuration file was read succesfully!")
+	}
+	private_key := viper.Get("credentials.aws_secret_access_key")
+	public_key := viper.Get("credentials.aws_access_key_id")
```

Comment:
I understand that it is more comfortable for us to have these credentials on syncron's configuration, but if a user already has it under '.aws/credentials', or something similar, my guess is that they will trust that better.